Examples/VisionExamples/Examples/Basics/GettingStartedViewController.swift

```diff
@@ -0,0 +1,87 @@
+/**
+ This file is used in a public documentation to show basic configuration steps.
+ */
+
+// Need to include Vision functionality
+import MapboxVision
+// Need to include Vision AR functionality
+import MapboxVisionAR
+// Need to include Vision Safety functionality
+import MapboxVisionSafety
+
+class GettingStartedViewController: UIViewController {
+    
+    // MARK: - Private properties
```

Comment:
Seems like we can remove mark comments in the example without the loss of helpfulness :)